tests/integration/dropdown/test_search_value.py

```diff
@@ -0,0 +1,36 @@
+from multiprocessing import Value
+import dash
+from dash.dependencies import Input, Output, State
+import dash.testing.wait as wait
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def test_search_value(dash_duo):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [dcc.Dropdown(id="dropdown", search_value="something"), html.Div(id="output")]
+    )
+
+    call_count = Value("i", 0)
+
+    @app.callback(
+        Output("output", "children"), inputs=[Input("dropdown", "search_value")]
+    )
+    def update_output(search_value):
+        call_count.value += 1
+        return 'search_value="{}"'.format(search_value)
+
+    dash_duo.start_server(app)
+
+    # Get the inner input used for search value.
+    input_ = dash_duo.find_element("#dropdown").find_element_by_tag_name("input")
+
+    # callback gets called with initial input
+    wait.until(lambda: call_count.value == 1, timeout=1)
+
+    assert dash_duo.wait_for_text_to_equal("#output", 'search_value="something"')
```

Comment:
all the `wait_for_*` APIs are some sort of assertions, so the `assert` here is not needed

Reply:
😈  who's the @byronz who wrote this line (in the test I pointed @inytar to as an example) that does exactly the same https://github.com/plotly/dash-core-components/blame/474f196d285e83cbe750bbbab998b220a0d61ee2/tests/integration/input/test_input_and_state.py#L38

Reply:
🦈  
that's why I didn't `suggest` but just points out it with a guilty conscience

Reply:
must be 🍻